src/generic/DirectSum.jl

```diff
@@ -48,10 +48,10 @@ function show(io::IO, N::DirectSumModule{T}) where T <: RingElement
    print(IOContext(io, :compact => true), base_ring(N))
 end
 
-function show(io::IO, N::DirectSumModule{T}) where T <: FieldElement
-   print(io, "Subspace over ")
-   print(IOContext(io, :compact => true), base_ring(N))
-end
+#function show(io::IO, N::DirectSumModule{T}) where T <: FieldElement
```

Comment:
Can we remove rather than comment out code. It's in GitHub if it needs to be retrieved.